app/controllers/api/proxy_rules_controller.rb

```diff
@@ -57,19 +46,25 @@ def authorize!
     provider_can_use!(:independent_mapping_rules)
   end
 
-  def find_proxy
-    @proxy = @service.proxy
+  helper_method :destroy_rule_path
+  def destroy_rule_path(service, rule)
+    admin_service_proxy_rule_path(service, rule)
   end
 
-  def find_proxy_rule
-    @proxy_rule = @proxy.proxy_rules.find(params[:id])
+  helper_method :edit_rule_path
+  def edit_rule_path(service, rule)
```

Comment:
Same as https://github.com/3scale/porta/pull/1072/files#r313297934, but let's keep the conversation only there 😄